### ISSUE TITLE
Placeholder for total stage number.

### ISSUE THREAD

**Issue by DateInJapanese** [PikaMug/Quests#1400]
**What does your feature do?**
It's a new placeholder to display the total amount of stages for the specified quest.
%quests_player_max_stage_number_<quest>%

Reason why I went with 'player_max_stage_number', instead of 'total_stage_amount' is because;
- The 'player' part is in every placeholder currently made, it doesn't make full sense here since it's per quest and not per player. Feel free to remove it of course, though it might be confusing considering it'd be only one then.
- max_stage_number because the placeholder would be similar to current_stage_number, making it easier to remember, allowing it to make sense.

**How would people use it?**
Admins could use to display the players progress through a stage more easily, not having to be afraid to edit the stage amount, and ending up having to re-write all their displays.
Also if they want to display the quest progress percentage-wise they can do it more easily with calculations.

Basically, it's a nice-to-have time-saving placeholder.

**Comment by PikaMug** [comment]
Thanks for the suggestion. All expansion-related requests should now be posted in https://github.com/PikaMug/PlaceholderExpansion-Quests/issues
